RocketCastTests/XMLParserTests.swift

```diff
@@ -0,0 +1,99 @@
+//
+//  XMLParserTests.swift
+//  RocketCast
+//
+//  Created by James Park on 2016-09-11.
+//  Copyright © 2016 UBCLaunchPad. All rights reserved.
+//
+
+import Foundation
+
+import XCTest
+@testable import RocketCast
+
+class XMLParserTests: XCTestCase {
+    
+    let episode1:[String:String] =
+        [ "title" : "Quantum Superposition",
+          "description": "exploring states of particles",
+          "author" : "Jon Mercer",
```

Comment:
Let's have this stuff in constants when we find that we have to copy and paste three times